html-forms/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1><strong>Coding Form!</strong></h1>
+    <div id="email-input">
+      <form action="index.html">
+        <label for="email">Email:
+          <div>
+            <input id="email" type="text">
+          </div>
+        </label>
+      </form>
+    </div> <!--end of email input-->
+    <div id="reason-for-coding">
+      <form action="index.html">
+        <label>Why is coding cool?</label>
+        <div>
+          <select name="reason">
+            <option value="option">Select an option</option>
+            <option value="interesting">Because it's interesting</option>
+            <option value="fun">Because it's fun</option>
+            <option value="powerful">Because it's powerful</option>
+          </select>
+        </div>
+      </form>
```

Comment:
You should not have an individual `form` element to enclose each form control elements, all the form control elements for this form should be inside of one `form` element.